src/components/NFTPage/PriceSection.vue

```diff
@@ -112,6 +121,27 @@ export default {
     handleClickCollect() {
       this.$emit('collect');
     },
+    handleClickSell() {
+      logTrackerEvent(
+        this,
+        'NFTSellButton',
+        'NFTSellButtonHandleClickSell',
+        '',
+        1
+      );
+    },
+    handleMouseEnterSell() {
+      if (!this.isSellHoverLogSend) {
+        logTrackerEvent(
+          this,
+          'NFTSellButton',
+          'NFTSellButtonHandleMouseEnterSell',
+          '',
+          1
+        );
+        this.isSellHoverLogSend = true;
+      }
+    },
```

Comment:
Suggest to emit `'click-sell'` and `'hover-sell'` event for details page to log the event